### PR TITLE
Add docs for citus_schema_move

### PR DESCRIPTION
**Pull request by onurctirtir**

### CODE REVIEW

File: develop/api_udf.rst [hunk 1]
```diff
@@ -67,6 +67,42 @@ Converting three different distributed schemas, back into regular schemas.
 
 For more examples, see :ref:`microservices_tutorial`.
 
+.. _citus_schema_move:
+
+citus_schema_move
+$$$$$$$$$$$$$$$$$$$$$$$$$
```
Comment (JelteF) [inline develop/api_udf.rst:73]: nit: let's make the underline as long as the name

```suggestion
citus_schema_move
$$$$$$$$$$$$$$$$$
```

File: develop/api_udf.rst [hunk 2]
```diff
@@ -67,6 +67,42 @@ Converting three different distributed schemas, back into regular schemas.
...
+
+This function moves a distributed schema from one node to another.
+
+There are two ways to move a distributed schema: blocking or nonblocking. The blocking approach means that during the move all modifications to the tables in the schema are paused. The second way, which avoids blocking writes, relies on Postgres 10 logical replication.
+
+Arguments
+**********
```
Comment (JelteF) [inline develop/api_udf.rst:80]:
```suggestion
Arguments
*********
```

File: develop/api_udf.rst [hunk 3]
```diff
@@ -67,6 +67,42 @@ Converting three different distributed schemas, back into regular schemas.
...
+  * ``force_logical``: Use logical replication even if the all the tables in the schema don't have a replica identity. Any concurrent update/delete statements to the tables in the schema will fail during replication.
+  * ``block_writes``: Use COPY (blocking writes) for the tables in the schema lacking primary key or replica identity.
+
+Return Value
+************
+
+N/A
+
+Example
+********
```
Comment (JelteF) [inline develop/api_udf.rst:100]:
```suggestion
Example
*******
```

File: develop/api_udf.rst [hunk 4]
```diff
@@ -67,6 +67,42 @@ Converting three different distributed schemas, back into regular schemas.
...
+
+.. code-block:: postgresql
+
+    SELECT citus_schema_move('schema-name'::regnamespace;, 'from_host', 5432, 'to_host', 5432);
```
Comment (JelteF) [inline develop/api_udf.rst:104]:
```suggestion
    SELECT citus_schema_move('schema-name', 'to_host', 5432);
```

File: develop/api_udf.rst [hunk 5]
```diff
@@ -67,6 +67,42 @@ Converting three different distributed schemas, back into regular schemas.
...
+$$$$$$$$$$$$$$$$$$$$$$$$$
+
+This function moves a distributed schema from one node to another.
+
+There are two ways to move a distributed schema: blocking or nonblocking. The blocking approach means that during the move all modifications to the tables in the schema are paused. The second way, which avoids blocking writes, relies on Postgres 10 logical replication.
+
+Arguments
+**********
+
+**schema_id:** Oid of the distributed schema to be moved.
```
Comment (JelteF) [inline develop/api_udf.rst:82]:
```suggestion
**schema_id:** Oid of the distributed schema to be moved. If you provide the name of the schema this as a string literal, this string is automatically casted to the oid.
```

**Review by JelteF — CHANGES_REQUESTED** [review]

File: develop/api_udf.rst (hunk 5 above)
Comment (onurctirtir) [reply develop/api_udf.rst:82]:
```suggestion
**schema_id:** Oid of the distributed schema to be moved. If you provide the name of the schema as a string literal, this string is automatically casted to the oid.
```

**Review by JelteF — APPROVED** [review]

**Pull request merged**